cs251tk/toolkit/__main__.py

```diff
@@ -139,18 +139,30 @@ def main():
     if assignments:
         available_specs = set(assignments)
 
+        if ci:
+            ignored_specs = open('.cs251tkignore').read().split('\n')
```

Comment:
Trim each line, too. 

1. Use a "with" block, so the file descriptor gets automatically closed
2. I think there's a .splitlines method (maybe .readlines()?) that handles both \n and \r\n
3. Be sure to trim each line